rpki-validator/src/main/java/net/ripe/rpki/validator3/background/Throttled.java

```diff
@@ -44,85 +43,122 @@
  * The purpose of this class is to be able to execute an arbitrary action
```

Comment:
This might makes it more obvious to library user: " to execute an arbitrary action having the same action key not more often than a certain interval", and in the example illustrate that "action1" is actually "action1Key".